registry/storage/driver/s3-aws/s3_test.go

```diff
@@ -36,6 +36,7 @@ func init() {
 	objectACL := os.Getenv("S3_OBJECT_ACL")
 	root, err := ioutil.TempDir("", "driver-")
 	regionEndpoint := os.Getenv("REGION_ENDPOINT")
+	sessionToken := os.Getenv("AWS_SESSION_TOKEN")
```

Comment:
How are the other environment variables provided to the test? You can generate the session token through [AWS Security Token Service](http://docs.aws.amazon.com/STS/latest/APIReference/Welcome.html).

I'll test it myself tomorrow.